docs/conf_extlinks.py

```diff
@@ -1,27 +1,28 @@
 # Use this for mapping to external links
 extlinks = {
-    "pynwb-docs": ("https://pynwb.readthedocs.io/en/stable/", "%s"),
-    "matnwb-src": ("https://github.com/NeurodataWithoutBorders/matnwb", "%s"),
+    "pynwb-docs": ("https://pynwb.readthedocs.io/en/stable/%s", "%s"),
+    "matnwb-src": ("https://github.com/NeurodataWithoutBorders/matnwb/%s", "%s"),
```

Comment:
sphinx has warned about this for a while but now it's official - kind of annoying, but even hard-coded links require a `%s` symbol in the base URL